(DOCSP-39502): Consolidate Define an Object Model page (#3314)
mongodb/docs-realm: source/includes/api-details/kotlin/model-data/object-models-map-model-or-property-to-different-name-description.rst

To map a Kotlin class or property name in your code to a different in the
database: 

#. Use the 
   :kotlin-sdk:`@PersistedName <io.realm.kotlin.types.annotations/-persisted-name/index.html>`
   annotation on the Kotlin class or property. 
#. Specify a class or property ``name`` that you want persisted to 
   the database. 

**Map a Model Name**

In this example, ``Frog`` is the Kotlin class name used in the code 
throughout the project to perform CRUD operations, and ``Frog_Entity`` is the

> persisted name to used to store objects in a realm:

Reviewer: ```suggestion
persisted name to used to store objects in the on-device database:
```
